src/amber/run_macros/generate_asset_helpers.cr

```diff
@@ -0,0 +1,58 @@
+require "json"
+require "colorize"
+
+class AssetManifestBuilder
+  MANIFEST_PATH = File.expand_path("./config/webpack/manifest.json")
```

Comment:
One last comment. 

I don't know anything about webpack, is manifest.json a standardized format or is it specific to webpack? If so, that's okay, but should this class be `WebpackAssetManifestBuilder`?

Actually, now that I think about it, this class is more of a `Parser` than a `Builder`, isn't it?

Reply:
Yes.  I would say it is more of a parser.  A relic of grabbing it from Lucky ;)